src/elements/Button/Button.js

```diff
@@ -250,8 +251,11 @@ class Button extends Component {
     }
 
     if (label) {
-      const classes = cx('ui', baseClasses, 'button', className)
-      const containerClasses = cx('ui', labeledClasses, 'button', className)
+      let classes = cx('ui', baseClasses, 'button', className)
+      if (className && className.indexOf('disabled') !== -1) {
+        classes = cxd(classes, { disabled: false })
```

Comment:
Yep, you're right. I thought the className checking must be added because there was className used in the example provided in #1026.